lib/irb/command/copy.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module IRB
+  module Command
+    class Copy < Base
+      category "Workspace"
+      description "Copy command output to clipboard"
+
+      help_message(<<~HELP)
+        Usage: copy (command)
+      HELP
+
+      def execute(arg)
+        value = irb_context.workspace.binding.eval(arg)
+        output = irb_context.inspect_method.inspect_value(value, colorize: false)
+
+        if clipboard_available?
+          copy_to_clipboard(output)
+        else
+          warn "System clipboard not found"
+        end
+      rescue StandardError => e
+        warn "Error: #{e}"
+        warn "Is IRB.conf[:COPY_COMMAND] set to a bad value?"
```

Comment:
Currently, `copy typo` will show this COPY_COMMAND warning message.
I think this message should be placed inside `def copy_to_clipboard`

Reply:
Updated! Here's how it works now:

```
λ  irb → λ git sin.add-copy-command → ./bin/console
irb(main):001> copy typo
Error: undefined local variable or method `typo' for main
irb(main):002> copy 5
Copied to system clipboard
irb(main):003> IRB.conf[:COPY_COMMAND] = 'lulz'
=> "lulz"
irb(main):004> copy 5
No such file or directory - lulz
Is IRB.conf[:COPY_COMMAND] set to a bad value?
irb(main):005>
```